seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1691,91 +1695,6 @@
       "challengeType": 0
     },
     {
-      "id": "bad87fee1348bd9aedf08818",
```

Comment:
The deleted challenge introduced the `alt` attribute. In this PR, I introduce the `alt` attribute at the same time the `img` element is introduced in **"Add Images to your Website"**. 

This offers best practice with regards to accessibility. 
RE #4656